addon/components/pixi-canvas.js

```diff
@@ -7,19 +7,30 @@ const {
 } = Ember;
 
 export default Component.extend({
-  pixiRenderer: computed('width', 'height', function() {
+  pixiRenderer: computed(function() {
     let { width, height } = this.getProperties('width', 'height');
 
     return PIXI.autoDetectRenderer(width, height);
   }),
 
+  resizePixiRenderer: Ember.observer('width', 'height', function() {
```

Comment:
Observers should generally be avoided in Ember. Can you make `width` and `height` writable computed properties instead?

Reply:
Hey :)
I changed the properties, not sure about the tests.